rotkehlchen/globaldb/upgrades/v11_v12.py

```diff
@@ -27,4 +34,21 @@ def _create_new_tables(write_cursor: 'DBCursor') -> None:
             """,
         )
 
+    @progress_step('Reset caches')
+    def _reset_caches(write_cursor: 'DBCursor') -> None:
+        write_cursor.execute(
+            'DELETE FROM general_cache WHERE key IN (?, ?, ?, ?)',
+            (
+                CacheType.VELODROME_POOL_ADDRESS.serialize(),
+                CacheType.VELODROME_GAUGE_ADDRESS.serialize(),
+                CacheType.AERODROME_POOL_ADDRESS.serialize(),
+                CacheType.AERODROME_GAUGE_ADDRESS.serialize(),
+            ),
+        )
+
+        write_cursor.execute(
+            'UPDATE unique_cache SET value=?, last_queried_ts = ? WHERE key = ?',
+            ('0', 0, CacheType.CURVE_LENDING_VAULTS.serialize()),
+        )
```

Comment:
nitpick: I would add a comment mentioning that this would trigger a refresh of the info to query new information